var/spack/repos/builtin/packages/py-openmc/package.py

```diff
@@ -38,7 +42,7 @@ class PyOpenmc(PythonPackage):
         )
 
     depends_on('git', type='build')
-    depends_on('python@3.5:', type=('build', 'run'))
+    depends_on('python@3.6:', type=('build', 'run'))
```

Comment:
The dependency here should be conditional on the version:
```suggestion
    depends_on('python@3.6:', type=('build', 'run'), when='@0.13:')
    depends_on('python@3.5:', type=('build', 'run'), when='@:0.12')
```